TileFormats/Instanced3DModel/README.md

```diff
@@ -98,19 +112,54 @@ In either case, `header.gltfByteLength` contains the length of the glTF field in
 
 The `instances` field immediately follows the `glTF` field (which may be omitted when `header.gltfByteLength` is `0`).
 
-The `instances` field contains `header.instancesLength` of tightly packed instances.  Each instance has three fields:
+The `instances` field contains `header.instancesLength` of tightly packed instances.  Each instance has the following fields:
+
+| Field name | Data type | Description | Required |
+| --- | --- | --- | --- |
+| `position.x` | `uint16` | The x-coordinate in quantized cartesian coordinates. | `yes` |
+| `position.y` | `uint16` | The y-coordinate in quantized cartesian coordinates. | `yes` |
+| `position.z` | `uint16` | The z-coordinate in quantized cartesian coordinates. | `yes` |
+| `v1.x` | `uint` | The first component of the oct-encoded vector `v1`. | `yes` |
+| `v1.y` | `uint` | The second component of the oct-encoded vector `v1`. | `yes` |
+| `v2.x` | `uint` | The first component of the oct-encoded vector `v2`. | `yes` |
+| `v2.y` | `uint` | The second component of the oct-encoded vector `v2`. | `yes` |
+| `batchId` | `uint16` | ID in the range `[0, length of arrays in the Batch Table)`, which indicates the corresponding properties. | `if header.batchTableByteLength > 0`
+
+### X, Y, and Z for Translation
+
+`x`, `y`, and `z` are stored as `uint16` positions in the quantized instance region defined by the `origin` and `span` fields in the header.
+
+**Figure 2:** The instance region defined by `origin` and `span`
+
+![](figures/instance-region.png)
+
+Transforming `position` in instance region space to `position_w` in world space can be done using the formula:
+
+`position_w` = `origin` + [`position` * `span` / (`2^16-1`)].
+
+### V1, and V2 for Rotation
+
+`v1` and `v2` are stored as two components in oct-encoded format as described in
+[*A Survey of Efficient Representations of Independent Unit Vectors* by Cigolle et al.](http://jcgt.org/published/0003/02/01/)
+The [AttributeCompression](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/AttributeCompression.js)
+module in Cesium contains an implementation for encoding and decoding vectors in this fashion.
+
+In their decoded form, `v1` and `v2` establish a new orthonormal basis, effectively rotating the coordinate system.
+
+The `y` and `x` vectors in the natural basis are transformed to map onto `v1` and `v2` respectively.
```

Comment:
I'll change this to read as `right`/`up`/`forward` which seems to be the common notation.